gravitee-apim-gateway/gravitee-apim-gateway-services/gravitee-apim-gateway-services-sync/src/main/java/io/gravitee/gateway/services/sync/synchronizer/AbstractSynchronizer.java

```diff
@@ -48,7 +49,7 @@ public abstract class AbstractSynchronizer extends AbstractService<AbstractSynch
 
     @Autowired
     @Qualifier("syncExecutor")
-    protected ExecutorService executor;
+    protected ThreadPoolExecutor executor;
```

Comment:
For my personal knowledge, why do we use this class instead of `ExecutorService`

Reply:
Because it offers access to useful methods compared to the `ExecutorService`. So instead of casting everywhere, I preferred to directly use the right class.